src/app/forms/validators/date/date.validators.ts

```diff
@@ -29,8 +29,7 @@ export class DateValidators {
 
   private static validateTime(time: string, label: string | undefined) {
     const [hours, minutes] = time.split(':');
-
-    if (!hours || !minutes) {
+    if (hours === '00' || minutes == '00') {
```

Comment:
what if the test is done at 00:00? That is a valid time no?

Reply:
Is this not also implying you can't type 00 in the minutes box at all meaning all tests that are conducted on the hour will be invalid times?